mods/action.js

```diff
@@ -181,6 +181,13 @@ function buildEditResponse(res) {
 }
 
 ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
+    if (/\.test\.local$/.test(req.params.domain)) {
```

Comment:
I'm pretty sure you looked into this before, so excuse me asking again: what stops us from using http://en.wikipedia.beta.wmflabs.org/wiki/Main_Page for tests in general? That could potentially avoid the need for hacks like this, and additionally avoid running into the edit limits on enwiki when running tests (yes, sometimes Travis is fast....).